reconcile/utils/terrascript_aws_client.py

```diff
@@ -3994,6 +3996,51 @@ def populate_tf_resource_elasticsearch(self, spec):
             "internal_user_database_enabled", False
         ):
             master_user = security_options["master_user_options"]
+            secret_values = {"name": identifier}
+            if provider:
+                secret_values["provider"] = provider
+            aws_secret_resource = aws_secretsmanager_secret(identifier, **secret_values)
```

Comment:
Similar concern as above: tf secret resources can be created by multiple integrations within the same account, i.e., there is a chance identifiers collide w/o prefixing. But maybe in this case this concern is too far fetched.